README.md

```diff
@@ -340,6 +340,7 @@ A curated list of amazingly awesome awesomeness.
 	- XMPP
 		- [Ejabberd](https://github.com/shantanu-deshmukh/awesome-ejabberd)
 	- [Typography](https://github.com/Jolg42/awesome-typography)
+	- [Coronavirus](https://github.com/soroushchehresa/awesome-coronavirus)
```

Comment:
The whole list is in alphabetical order. Please move your list to the right place (just after line 19).

Reply:
@KleoPetroff Fixed.